tests/schema/test_sync_dataloader.py

```diff
@@ -0,0 +1,143 @@
+from promise import Promise
+from promise.dataloader import DataLoader
+
+import strawberry
+from strawberry.schema.execute_context import ExecutionContextWithPromise
+
+
+def test_batches_correct():
+    load_calls = []
+
+    class TestDataLoader(DataLoader):
+        def batch_load_fn(self, keys):
+            load_calls.append(keys)
+            return Promise.resolve(keys)
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def get_id(self, info, id: str) -> str:
+            return info.context["dataloader"].load(id)
+
+    schema = strawberry.Schema(query=Query)
+    result = schema.execute_sync(
+        """
+        query {
+            id1: getId(id: "1")
+            id2: getId(id: "2")
+        }
+    """,
```

Comment:
Looks like the indentation got messed up here?

Reply:
Appears to be an issue with a couple more `"""` strings below

Reply:
It's how black formats it 🤷 I think it looks weird too.

Reply:
Strange that black would format `"""` strings, seeing that the whitespace can be an important part of them :thinking: